x-pack/plugin/core/template-resources/src/main/resources/kibana-reporting-template.json

```diff
@@ -0,0 +1,168 @@
+{
+  "index_patterns": [".kibana-reporting*"],
+  "priority": 100,
+  "data_stream": {
+    "hidden": true
+  },
+  "allow_auto_create": true,
+  "template": {
+    "settings": {
+      "number_of_shards": 1,
+      "auto_expand_replicas": "0-1"
+    },
+    "mappings": {
+      "properties": {
+        "meta": {
+          "properties": {
+            "objectType": {
+              "type": "text",
+              "fields": {
+                "keyword": {
+                  "type": "keyword",
+                  "ignore_above": 256
+                }
+              }
+            },
+            "layout": {
+              "type": "text",
+              "fields": {
+                "keyword": {
+                  "type": "keyword",
+                  "ignore_above": 256
+                }
+              }
+            },
+            "isDeprecated": {
+              "type": "boolean"
+            }
+          }
+        },
+        "migration_version": {
+          "type": "keyword"
+        },
+        "jobtype": {
+          "type": "keyword"
+        },
+        "payload": {
+          "type": "object",
+          "enabled": false
+        },
+        "priority": {
+          "type": "byte"
+        },
+        "timeout": {
+          "type": "long"
+        },
+        "process_expiration": {
+          "type": "date"
+        },
+        "created_by": {
+          "type": "keyword"
+        },
+        "created_at": {
+          "type": "date"
+        },
+        "started_at": {
+          "type": "date"
+        },
+        "completed_at": {
+          "type": "date"
+        },
+        "attempts": {
+          "type": "short"
+        },
+        "max_attempts": {
+          "type": "short"
+        },
+        "kibana_name": {
+          "type": "keyword"
+        },
+        "kibana_id": {
+          "type": "keyword"
+        },
+        "status": {
+          "type": "keyword"
+        },
+        "parent_id": {
+          "type": "keyword"
+        },
+        "output": {
+          "type": "object",
+          "properties": {
+            "error_code": {
+              "type": "keyword"
+            },
+            "chunk": {
+              "type": "long"
+            },
+            "content_type": {
+              "type": "keyword"
+            },
+            "size": {
+              "type": "long"
+            },
+            "content": {
+              "type": "object",
+              "enabled": false
+            }
```

Comment:
I noticed after creating a CSV report, Elasticsearch logged that it updated mappings for this index.

I found that under `output`, there are 2 other fields needed: `output.csv_contains_formulas` and `output.max_size_reached`
```json
        "csv_contains_formulas": {
          "type": "boolean"
        },
        "max_size_reached": {
          "type": "boolean"
        },
```

Reply:
Okay, I pushed a commit to add these two fields.

Reply:
Thanks! I just tested again and found there is another field, `output.warnings` which is just a text field.

```
            "warnings": {
              "type": "text",
            }
```